nginx-proxy.conf

```diff
@@ -1,7 +1,7 @@
 include /etc/nginx/api-proxy.conf;
 
 location / {
-    rewrite (?i)\.(jp(e)?g|gif|png|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$host$request_uri;
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip|js|css)$ https://static.zooniverse.org/$host$request_uri;
```

Comment:
I thought the security scanner issue was because the assets were being hosted cross-domain, so static.zooniverse.org would be fine. But if it's because the rewrite directive makes for a 302 and doesn't actually proxy the zooniverse URL, then we'd have the same problem. Is it worth then going back and changing back all the ones that I changed along the way, but haven't heard any complaints about? Or leaving them and making this one separate as a precaution?

Reply:
>  thought the security scanner issue was because the assets were being hosted cross-domain, so static.zooniverse.org would be fine.

The security scanner is a black box. In #222 we didn't know why it determined the JS loading on the 3rd party domain was flagged as phising, however we have a very good idea that it was linked to loading on a different domain. A subdomain is considered a different domain for the Same Origin Policy so may be treated differently in the security scanners, we really can't tell. We can use the the TrendMicro URL testing site to see if a known URL fails a security scanner check, https://www.zooniverse.org/talk/17/1942061?comment=3162303

Re-reading #222 i think my point about the CDN is probably key here and I've changed my mind. 
> On a case by case basis in future if the host domain has it's own CDN use the proxy pass technique. Alternatively use the static.zooniverse.org/$host domain as it has a CDN to distribute important web assets for us on our own trusted subdomain.

So in this particular fallback case we can first see how redirecting on the CDN (AFD) hosted static domain work out as that gains us the CDN benefits. However if we encounter any issues we can switch back to serving the JS (or whichever assets are impacted) over the host domain via the custom proxy pass location block. 

FWIW - if the domain in question has it's own CDN, e.g. www.zooniverse.org then i suggest we replicate the previous setup and serve the JS & CSS assets on that domain to avoid any of the issues outlined above.

So feel free to mark this as resolved and we can revisit if needed and we have a reference JS & CSS proxy pass implementation in the www.zooniverse.org server block for future use.